modules/platforms/cpp/ignite/client/sql/sql_statement.h

```diff
@@ -132,6 +134,26 @@ class sql_statement {
      */
     void properties(std::initializer_list<std::pair<const std::string, primitive>> val) { m_properties = val; }
 
+    /**
+     * Gets the Timezone ID.
+     *
+     * @return Timezone ID.
+     */
+    [[nodiscard]] const std::string &timezone_id() const { return m_timezone_id; }
+
+    /**
+     * Sets the Timezone ID.
+     *
+     * Examples: "America/New_York", "UTC+3".
+     * Affects time-related SQL functions (e.g. @c CURRENT_TIME) and string literal conversions
+     * (e.g. <tt>TIMESTAMP WITH LOCAL TIME ZONE '1992-01-18 02:30:00.123'</tt>).
+     *
+     * If left empty, defaults to server time zone.
```

Comment:
Empty or null?

Reply:
Empty, only pointers in C++ can be `null`.